src/proposals/20240905/SparkEthereum_20240905.t.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import 'src/SparkTestBase.sol';
+
+import { IKillSwitchOracle } from 'lib/sparklend-kill-switch/src/interfaces/IKillSwitchOracle.sol';
+
+interface IOracle {
+    function latestAnswer() external view returns (int256);
+}
+
+contract SparkEthereum_20240905Test is SparkEthereumTestBase {
+
+    address internal constant WETH_ORACLE_OLD =   0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419;
+    address internal constant WSTETH_ORACLE_OLD = 0x8B6851156023f4f5A66F68BEA80851c3D905Ac93;
+    address internal constant RETH_ORACLE_OLD =   0x05225Cd708bCa9253789C1374e4337a019e99D56;
+    address internal constant WEETH_ORACLE_OLD =  0x1A6BDB22b9d7a454D20EAf12DB55D6B5F058183D;
+
+    address internal constant WETH_ORACLE = 0xf07ca0e66A798547E4CB3899EC592e1E99Ef6Cb3;
+    address internal constant WSTETH_ORACLE = 0xf77e132799DBB0d83A4fB7df10DA04849340311A;
+    address internal constant RETH_ORACLE = 0x11af58f13419fD3ce4d3A90372200c80Bc62f140;
+    address internal constant WEETH_ORACLE = 0x28897036f8459bFBa886083dD6b4Ce4d2f14a57F;
```

Comment:
Alignment should be consistent. I don't mind much on the formatting, but you should stick with the same style.

Reply:
Align equals for all